### DISCUSSION TITLE
Single Product belongs to many collections

### DISCUSSION THREAD

**Discussion by ashkanyadegari** [medusajs/medusa#3265] [Feature Requests]
Hi community!

I remember there was a PR/Issue on this matter, however there was a request to submit ideas in the discussions board here.

Medusa is amazing! I think it can even be better.

Current Status:

In Medusa you can only attach a product to a single collection, however in many e-commerce use cases companies will assign a product to multiple collections. 

Request:

Allow a product to be attached to more than one collection.

Kind regards,

**Comment by srindom** [comment]
@ashkanyadegari thanks for submitting this - we will very soon be launching Product Categories which will allow you to place a product in multiple categories and also nest category relationships. 

@riqwan has been leading the work and we look forward to see it in use 👌

**Comment by voxsar** [comment]
Was this implemented?
